Nodejs/Product/Nodejs/ProjectTemplates/VuejsApp/VuejsApp.njsproj

```diff
@@ -45,18 +45,17 @@
     <Folder Include="src\components\" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="babel.config.js" />
-    <Compile Include="src\main.js" />
+    <Content Include="babel.config.js" />
```

Comment:
Does this template already have an appropriate package.json file?

Reply:
It does, but a change on the njsproj is required. Commit sent.